scripts/build_genome_data.sh

```diff
@@ -3,6 +3,8 @@
 set -e
 
 if [[ "$#" -lt 2 ]]; then
+  echo
+  echo "ACTIVATE PIPELINE'S CONDA ENVIRONMENT BEFORE RUNNING THIS SCRIPT!"
```

Comment:
Can simplify to remove the `echo` above:
```suggestion
  echo -e "\nACTIVATE PIPELINE'S CONDA ENVIRONMENT BEFORE RUNNING THIS SCRIPT!"
```

Reply:
thanks, but I will keep using empty `echo` for better code readability.

Reply:
Cool, yeah there is definitely a tradeoff here between brevity and readability.